sdks/python/apache_beam/io/gcp/bigquery_tools_test.py

```diff
@@ -451,7 +451,9 @@ def test_read_from_table(self):
     client.jobs.GetQueryResults.return_value = bigquery.GetQueryResultsResponse(
         jobComplete=True, rows=table_rows, schema=schema)
     actual_rows = []
-    with beam.io.BigQuerySource('dataset.table').reader(client) as reader:
+    with beam.io.BigQuerySource(
```

Comment:
Could you file a JIRA to update these to not depend on the deprecated source?

Reply:
+1. Any reason why we can use the new source here today ?

Reply:
I've filed the JIRA (BEAM-10951) - the tests in this class are using `BigQuerySource`-specific functions. The new sources do not implement the `reader` function, which we use mainly to support BQSource in direct runner. The tests in this class should be deleted if BQSource goes away.